src/Event.php

```diff
@@ -88,6 +88,20 @@ class Event implements PingableInterface
      */
     protected $timezone;
 
+    /**
+     * Datetime or time since the task is evaluated and possibly executed only for display purposes.
+     *
+     * @var \DateTime|string|null
+     */
+    protected $from = null;
```

Comment:
Add union type (`protected \DateTime|string|null $from = null;`) in both cases.

Reply:
Done